### PR TITLE
[Hono] Add the Let's Encrypt root cert to trusted-certs.pem

### PR DESCRIPTION
**Pull request by b-abel**
Adds the root certificate (https://letsencrypt.org/certs/isrgrootx1.pem) of Let's Encrypt to the example trust store. This allows deploying setups where the example certificates are used together with certificates issued by the Let's Encrypt CA.

### CODE REVIEW

File: charts/hono/example/create_certs.sh [hunk 1]
```diff
@@ -105,7 +105,7 @@ openssl req -config ca_opts -reqexts intermediate_ext -new -key $DIR/ca-key.pem
 
 echo ""
 echo "creating PEM trust store ($DIR/trusted-certs.pem) containing CA certificate"
-cat $DIR/ca-cert.pem $DIR/root-cert.pem > $DIR/trusted-certs.pem
+cat $DIR/ca-cert.pem $DIR/root-cert.pem lets-encrypt-root-cert.pem > $DIR/trusted-certs.pem
```
Comment (sophokles73) [inline charts/hono/example/create_certs.sh:108]: maybe we can download the cert from the Let's Encrypt site instead? That would relief us from having to maintain a copy in the repo ...

**Review by calohmn — COMMENTED** [review]
Shouldn't also the intermediate certificate be added to `trusted-certs.pem`, i.e. R3 from [here](https://letsencrypt.org/certificates/)?

**Comment by b-abel** [comment]
The script downloads now the LE root and intermediate certificate and adds them to the trust store.

**Review by sophokles73 — APPROVED** [review]
LGTM

**Comment by sophokles73** [comment]
@calohmn WDYT?

**Review by calohmn — APPROVED** [review]
LGTM

**Pull request merged**